tools/ui

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+if [ "$1" == "run" ]; then
+    npm run --prefix ui start
+elif [ "$1" == "buildtoae" ]; then
+    npm run --prefix ui build
+    mkdir -p app/resources/static/ui_bundles
+    mv ui/dist/*-bundle.js app/resources/static/ui_bundles/
+    mv ui/dist/bundle.css app/resources/static/ui_bundles/bundle.css
+    cp -r --no-clobber ui/static/static/* app/resources/static/
```

Comment:
Out of scope of this pull request but just curious, why are icons in a directory ui/static/static/icons? Why not ui/static/icons?

Reply:
Well, the Webpack dev server expects static files to be in a static directory, and serves whatever's in there from the root directory. But we also serve static files from the /static directory (and the HTML the React app produces needs to point there, because it's what will happen in prod). So, if we put the static files in a static directory within Webpack's static directory, it'll serve files from the same URLs prod uses.
I think there's a way around it by setting up a proxy (as we do with /d to have requests sent to the Django app at port 8000), but I haven't been able to get it set up right yet.

Reply:
I see. It isn't too bad, but maybe good to leave comment somewhere (in static/README.md?) to explain the reason.